server/src/dekorator/appMedNavDekoratoren.ts

```diff
@@ -1,17 +1,24 @@
 import { injectDecoratorServerSide } from '@navikt/nav-dekoratoren-moduler/ssr';
-import { Response } from 'express-serve-static-core';
+import { Request, Response } from 'express-serve-static-core';
 
 import { getEnv } from '../paths/miljo';
 
 async function getNavdekoratoren(
     indexFilepath: string,
+    req: Request,
     res: Response<any, Record<string, any>, number>,
 ): Promise<void> {
+    let innloggetPart = req.cookies['innlogget-part'];
+    if (['DELTAKER'].includes(innloggetPart)) {
+        innloggetPart = 'privatperson';
+    } else {
+        innloggetPart = 'arbeidsgiver';
+    }
```

Comment:
Hmm, kanskje jeg har misset noe. Men hvorfor sjekkes `innloggetPart` mot en array som inneholder "DELTAKER" og ikke bare en string?

```suggestion
    const innloggetPart = req.cookies['innlogget-part'] === 'DELTAKER' ? 'privatperson' : 'arbeidsgiver';
```